nmstate/README.md

```diff
@@ -1,25 +1,32 @@
 # NMState
 
-## Usage
-
 This is to deploy [NMState operator](https://docs.openshift.com/container-platform/4.9/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html) to OpenShift. The NMState Operator provides users with functionality to configure various network interface types, DNS, and routing on cluster nodes
```

Comment:
Possibly update to 4.14 docs? Or 'latest'?
https://docs.openshift.com/container-platform/latest/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html

Reply:
Great idea!  Done.